api/python/tests/test_cli.py

```diff
@@ -46,6 +48,42 @@ def test_push(self):
                 mocked_push.assert_called_once_with(name, registry=None, dest=None, message=None)
 
 
+@pytest.mark.parametrize(
+    'name, registry, expected_calls, expected_no_of_calls',
+    [
+        ('asah/gpt3', 's3://another-bucket', [], 1),
+        (
+            'quilt.yml',
+            None,
+            [
+                call('asah/gpt3', dest=None, dest_registry=None, path=None, registry='s3://some-bucket',
+                     top_hash=None),
+                call('akarve/lmnb1', dest=None, dest_registry=None, path=None, registry='s3://some-bucket',
+                     top_hash='c698234'),
+                call('asah/gpt3', dest=None, dest_registry=None, path=None, registry='s3://another-bucket',
+                     top_hash=None),
+                call('akarve/lmnb1', dest=None, dest_registry=None, path=None, registry='s3://another-bucket',
+                     top_hash='c698234'),
+                call('akarve/lmnb3', dest=None, dest_registry=None, path='sub/path', registry='s3://another-bucket',
+                     top_hash='c698234')
+            ],
+            5
+        )
+    ]
+)
+def test_cli_install(name, registry, expected_calls, expected_no_of_calls, set_temporary_working_dir):
```

Comment:
If I'm not missing something `set_temporary_working_dir` is an autouse fixture, so it's not needed here:

```suggestion
def test_cli_install(name, registry, expected_calls, expected_no_of_calls):
```